### PR TITLE
get rid of references to 'hpcugent' organisation after move to github.com/easybuilders in documentation

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@easybuilders/easybuild-docs-maintainers please review/merge?

File: docs/index.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
 EasyBuild documentation
 =======================
 
-Welcome to the documentation of `EasyBuild <https://hpcugent.github.io/easybuild>`_, a software build and installation framework
+Welcome to the documentation of `EasyBuild <http://easybuilders.github.io/easybuild>`_, a software build and installation framework
```
Comment (vanzod) [inline docs/index.rst]: @boegel Why did you drop the `https`?

File: docs/version-specific/Supported_software.rst [hunk 2]
```diff
@@ -4310,7 +4310,7 @@ version         toolchain
 
 EasyBuild is a software build and installation framework  written in Python that allows you to install software in a structured,  repeatable and robust way.
 
-*homepage*: http://hpcugent.github.com/easybuild/
+*homepage*: http://easybuilders.github.io/easybuild/
```
Comment (vanzod) [inline docs/version-specific/Supported_software.rst]: @boegel Redirect to `https` maybe?

File: docs/demos/scripts/review_pr.script [hunk 3]
```diff
@@ -1,6 +1,6 @@
 CLEAR
 ## EasyBuild is a build and installation framework for (scientific) software on HPC systems
-## http://hpcugent.github.io/easybuild/ - http://easybuild.readthedocs.io/
+## http://easybuilders.github.io/easybuild/ - http://easybuild.readthedocs.io/
```
Comment (vanzod) [inline docs/demos/scripts/review_pr.script]: @boegel Replace `http` with `https`?

File: docs/demos/scripts/configuring.script [hunk 4]
```diff
@@ -1,6 +1,6 @@
 CLEAR
 ## EasyBuild is a build and installation framework for (scientific) software on HPC systems
-## http://hpcugent.github.io/easybuild/ - http://easybuild.readthedocs.io/
+## http://easybuilders.github.io/easybuild/ - http://easybuild.readthedocs.io/
```
Comment (vanzod) [inline docs/demos/scripts/configuring.script]: @boegel Replace http with https?

File: docs/demos/scripts/bootstrap.script [hunk 5]
```diff
@@ -1,6 +1,6 @@
 CLEAR
 ## EasyBuild is a build and installation framework for (scientific) software on HPC systems
-## http://hpcugent.github.io/easybuild/ - http://easybuild.readthedocs.io/
+## http://easybuilders.github.io/easybuild/ - http://easybuild.readthedocs.io/
```
Comment (vanzod) [inline docs/demos/scripts/bootstrap.script]: @boegel Replace `http` with `https`?

File: docs/index.rst (hunk 1 above)
Comment (boegel) [inline docs/index.rst]: I was under the impression that https://easybuilders.github.io/easybuild/ just redirect to http://easybuilders.github.io/easybuild/, which is (of course) not the case...

Fixed in 233d423

File: docs/demos/scripts/bootstrap.script [hunk 6]
```diff
@@ -58,7 +58,7 @@ CLEAR
 ### STEP 1: download EasyBuild bootstrap script
 ###
 
-curl -O COPYPASTEhttps://raw.githubusercontent.com/hpcugent/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
+curl -O COPYPASTEhttps://raw.githubusercontent.com/easybuilders/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
```
Comment (vanzod) [inline docs/demos/scripts/bootstrap.script]: @boegel I think you missed a space between the protocol specification and `https`

Comment (boegel) [inline docs/demos/scripts/bootstrap.script]: No, this is correct, the `COPYPASTE` is a keyword that tells my patched version of `asciicinema` to copy-paste what follows. Adding a space after would also add a space in what is being copy-pasted.

Comment (vanzod) [inline docs/demos/scripts/bootstrap.script]: Ok, thanks for clarifying

**Comment by vanzod** [comment]
Thanks @boegel

File: docs/demos/scripts/bootstrap.script (hunk 6 above)
Comment (damianam) [inline docs/demos/scripts/bootstrap.script]: It certainly looks weird. Maybe it makes sense to add a comment right on top?

**Pull request merged**